syntax/neomake/qf.vim

```diff
@@ -0,0 +1,5 @@
+syntax clear
```

Comment:
Doesn't this clear other custom configs?
Is it required for resetting/reloading?